pkg/common/jobcontroller/jobcontroller.go

```diff
@@ -231,6 +261,24 @@ func (jc *JobController) SyncPdb(job metav1.Object, minAvailableReplicas int32)
 	return jc.KubeClientSet.PolicyV1beta1().PodDisruptionBudgets(job.GetNamespace()).Create(createPdb)
 }
 
+func (jc *JobController) DeletePodGroup(job metav1.Object) error {
+	kubeBatchClientInterface := kbutil.GetKubeBatchClientInterface()
+
+	//check whether podGroup exists or not
+	_, err := kubeBatchClientInterface.SchedulingV1alpha1().PodGroups(job.GetNamespace()).Get(job.GetName(), metav1.GetOptions{})
+	if err != nil && k8serrors.IsNotFound(err) {
+		return nil
+	}
+
+	log.Infof("Deleting PodGroup %s", job.GetName())
+
+	//delete podGroup
+	if err := kubeBatchClientInterface.SchedulingV1alpha1().PodGroups(job.GetNamespace()).Delete(job.GetName(), &metav1.DeleteOptions{}); err != nil {
```

Comment:
Nit: easier to read if this is split into 2 lines at the "if"

Reply:
Done